samples/MvvmSampleUwp/MvvmSampleUwp.csproj

```diff
@@ -388,55 +388,53 @@
     </Page>
```

Comment:
Isn't there a Xamarin head in main, so that needs updating too?

Reply:
Done in https://github.com/CommunityToolkit/MVVM-Samples/pull/89/commits/07a13d5c424212a78dd490674d92e05c60ceba8f 🙂